src/interpreter/index.ts

```diff
@@ -247,12 +247,14 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
                 file: "(internal)",
             },
         });
+
         let maybeCallback = this.evaluate(callbackVariable);
         if (maybeCallback.kind === ValueKind.Callable) {
             return maybeCallback;
         }
 
-        throw new NotFound(`${functionName} was not found in scope`);
+        console.warn(`Warning: "${functionName}" was not found in scope.`);
```

Comment:
Let me know if `console.warn()` is appropriate, here, or if `stderr.write()` would be better.

Reply:
I typically prefer `stderr.write()` so it doesn't appear in end-to-end test output (plus it's easily capturable/testable that way!)